pkg/actors/actors.go

```diff
@@ -1446,7 +1446,7 @@ func (a *actorsRuntime) getActorTypeMetadata(actorType string, migrate bool) (*A
 	} else {
 		retryErr = backoff.Retry(func() error {
 			metadataKey := constructCompositeKey("actors", actorType, "metadata")
-			resp, err := a.store.Get(&state.GetRequest{
+			resp, err := a.store.Get(context.TODO(), &state.GetRequest{
```

Comment:
Should we use context.Background() here as well?

Reply:
No, I think TODO here is correct. Because we will need to cascade the context, eventually.